engine/render/backend/api/CMakeLists.txt

```diff
@@ -1,5 +1,14 @@
-list(APPEND CHIRA_ENGINE_HEADERS
+if (${CHIRA_RENDER_BACKEND} MATCHES "OpenGL")
+    list(APPEND CHIRA_ENGINE_HEADERS
         ${CMAKE_CURRENT_LIST_DIR}/BackendGL.h)
-
-list(APPEND CHIRA_ENGINE_SOURCES
+    list(APPEND CHIRA_ENGINE_SOURCES
         ${CMAKE_CURRENT_LIST_DIR}/BackendGL.cpp)
+
+elseif (${CHIRA_RENDER_BACKEND} MATCHES "DirectX")
```

Comment:
Possible values for this variable should be listed in the root cmake

Reply:
will do!